app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -185,9 +186,15 @@ function Form(
   const isLabelEvent = modEventType === MOD_EVENTS.LABEL
   const isDivertEvent = modEventType === MOD_EVENTS.DIVERT
   const isMuteEvent = modEventType === MOD_EVENTS.MUTE
+  const isUnmuteEvent = modEventType === MOD_EVENTS.UNMUTE
   const isCommentEvent = modEventType === MOD_EVENTS.COMMENT
   const shouldShowDurationInHoursField =
     modEventType === MOD_EVENTS.TAKEDOWN || isMuteEvent
+  const isReportingMutedForSubject = isMuted(subjectStatus, true)
+  const shouldShowMuteReportingOnlyField =
+    isSubjetDid &&
```

Comment:
Not related to the work in this PR, but I noticed a typo in this variable name: `isSubjetDid` vs. `isSubjectDid`.